stringlength.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+)
+
+func main() {
+	stringOne := "merpflakes"
+	stringTwo := "   "
+	stringThree := ""
+
+	if len(strings.TrimSpace(stringOne)) == 0 {
```

Comment:
👍 
Could you create a separate function to get the string length and try to unit test it ? I'm really curious about unit testing in Go !